src/compiler/utils/patterns.ts

```diff
@@ -1,3 +1,5 @@
 export const whitespace = /[ \t\r\n]/;
 
 export const dimensions = /^(?:offset|client)(?:Width|Height)$/;
+
+export const sizing = /^contentRect|contentBoxSize|borderBoxSize|devicePixelContentBoxSize$/;
```

Comment:
The regex is incorrect I think, the words need to be surrounded like `^(a|b|c)$`

Reply:
It doesn't need parens, unless I want to get fancy (and less readable) with:

```js
/^contentRect|(?:content|border|devicePixelContent)BoxSize$/
```

I thought that got a bit too hard to read, but I can change it if you feel strongly about it.

Reply:
Resolved. @dummdidumm was right that we had to wrap a parentheses for the regex to properly work.